### DISCUSSION TITLE
an issue for Connect PetStoreApp and PetStoreService Together

### DISCUSSION THREAD

**Discussion by whitelee123** [chtrembl/azure-cloud#32] [Q&A]
![Screenshot 2023-11-19 at 22 02 21](https://github.com/chtrembl/azure-cloud/assets/44828996/9fe89d5f-41c3-4927-8269-7d8ef7288732)

I have set up the web app and AKS cluster. but got this error above. 

Could you help me figure it out plz.

**Comment by chtrembl** [comment]
Hi @whitelee123 thanks for reaching out, this was just fixed in this [commit](https://github.com/chtrembl/azure-cloud/commit/f18de8de9fe55304ed6c76209f14ad199d222401) and the details can be see in this [issue](https://github.com/chtrembl/azure-cloud/issues/43)
